ludwig/data/dataset/ray.py

```diff
@@ -74,8 +74,6 @@ def __init__(
     ):
         self.df_engine = backend.df_engine
         self.ds = self.df_engine.to_ray_dataset(df) if not isinstance(df, str) else read_remote_parquet(df)
-        if self.size == 0:
```

Comment:
I wonder if we should still warn the user in these cases, in case the full test dataset ended up being dropped for one reason or another when they didn't expect that.

Reply:
If we do, it should be in the preprocessing file below, though.

Reply:
Logging a warning when seems reasonable. Added!